CH-679-CH-739 changed review service

NatalliaBulyha opened the pull request:
jira[change logic in endpoint add new review] https://cozyhometeam.atlassian.net/browse/CH-739?atlOrigin=eyJpIjoiZWI3NzMwMTVjMGZmNDkyNzk0ODdjNjY1NThjM2U3ZDMiLCJwIjoiaiJ9

Commit:
CH-679-CH-739 changed review service

jira[change logic in endpoint add new review] https://cozyhometeam.atlassian.net/browse/CH-739?atlOrigin=eyJpIjoiZWI3NzMwMTVjMGZmNDkyNzk0ODdjNjY1NThjM2U3ZDMiLCJwIjoiaiJ9

DariaBoo commented on src/main/java/com/cozyhome/onlineshop/reviewservice/service/ReviewServiceImpl.java:
```diff
@@ -50,15 +48,12 @@ public List<ReviewResponse> getReviewsForProduct(String productSkuCode) {
-            log.error("[ON removeReviewById]:: You can only delete your own reviews.");
-            throw new AccessDeniedException("You can only delete your own reviews.");
+            log.error("[ON removeReviewById]:: Review with id {} doesn't exist.", reviewRemoveDto.getReviewId());
+            throw new AccessDeniedException(String.format("[ON removeReviewById]:: Review with id %s doesn't exist.", reviewRemoveDto.getReviewId()));
```
DataNotFoundException?

NatalliaBulyha replied:
done

Commit:
CH-679-CH-739 corrections as noted

jira[endpoint: change password in personal account] https://cozyhometeam.atlassian.net/browse/CH-738?atlOrigin=eyJpIjoiNWI0NTU5MDczMjYyNDA3MzhjYmM3MzUxYmNmYzY2N2IiLCJwIjoiaiJ9
